README.rst

```diff
@@ -44,6 +52,21 @@ Alternatively, you might want to use the `Moses <http://www.statmt.org/moses/>`_
 
     pip install sacremoses
 
+For torchtext 0.5 and below, ``sentencepiece``::
+
+    conda install -c powerai sentencepiece
+
+Building from source
+--------------------
+
+To build torchtext from source, you need ``git``, ``CMake`` and C++11 compiler such as ``g++``.::
+
+    git clone https://github.com/pytorch/text torchtext
+    cd torchtext
+    git submodule update --init --recursive
+    python setup.py clean install
```

Comment:
Please double check that the package is installed at global site-packages directory properly. You could try to `import torchtext` outside the `text` folder and see if it works properly.

Reply:
Oops, you are right. The installation does not work.
I think https://github.com/pytorch/text/pull/764 should solve this.